components/motor/gpio/motor_encoder.go

```diff
@@ -471,8 +484,10 @@ func (m *EncodedMotor) goForInternal(ctx context.Context, rpm, revolutions float
 	defer m.stateMu.Unlock()
 
 	if revolutions == 0 {
+		// Moving 0 revolutions is a special value meaning "move forever."
```

Comment:
this comment is correct, but i think the change making regulated true is wrong.
regulated means we're going to a position.
which is the opposite of the comment